libexec/klab-build-js

```diff
@@ -109,7 +108,7 @@ Object.keys(config.implementations)
   .forEach(alias => {
     const cname = config.implementations[alias].name
     const name = alias + "__exhaustiveness";
-    const {id, module, status} = makeInterabiExhaustiveness(alias, cname, config.contracts[cname])
+    const {id, module, status} = makeInterabiExhaustiveness(alias, cname, config.contracts[cname], config.get_proof_hash)
```

Comment:
I wonder if it wouldn't make sense to just pass a `config` argument to `makeInterabiExhaustiveness` instead of `contract` and `hasher`.

The main reason is that after a certain point adding more arguments makes the function less readable. But OTOH I also understand not wanting to pass huge global objects to a function and keeping its arguments lean and specific, so that `makeInterabiExhaustiveness` doesn't need to know anything about `config`.

As I wrote this, I realized that i prefer your solution, but leaving this comment here for discussion.